code/modules/mob/delays.dm

```diff
@@ -34,11 +34,11 @@
 // Constructor args are currently all the same, but placed here for ease of tuning.
 /client // Yep, clients are snowflakes.
 	// Walking speed is 7, as is grab speed.
-	var/datum/delay_controller/move_delayer    = new (1, 100) // /mob/delayNextMove()
+	var/datum/delay_controller/move_delayer    = new (1,10000) // /mob/delayNextMove()
 /mob
-	var/datum/delay_controller/click_delayer   = new (1, 100) // (Handled in Click())
-	var/datum/delay_controller/attack_delayer  = new (1, 100) // delayNextAttack()
-	var/datum/delay_controller/special_delayer = new (1, 100) // delayNextSpecial()
+	var/datum/delay_controller/click_delayer   = new (1,10000) // (Handled in Click())
```

Comment:
Yes, but you also had to edit them multiple times.
Should be a define. This is the exact reason to use a define.